core-feature-pack/common/src/main/resources/modules/system/layers/base/wildflyee/api/main/module.xml

```diff
@@ -20,35 +20,33 @@
 
 <module xmlns="urn:jboss:module:1.9" name="wildflyee.api">
     <dependencies>
-        <module name="javax.activation.api" export="true" optional="true" />
-        <module name="javax.annotation.api" export="true" optional="true" />
-        <module name="javax.batch.api" export="true" optional="true" />
-        <module name="javax.ejb.api" export="true" optional="true" />
-        <module name="javax.enterprise.api" export="true" optional="true" />
-        <module name="javax.enterprise.concurrent.api" export="true" optional="true" />
-        <module name="javax.inject.api" export="true" optional="true" />
-        <module name="javax.interceptor.api" export="true" optional="true" />
-        <module name="javax.json.api" export="true" optional="true" />
-        <module name="javax.jms.api" export="true" optional="true" />
-        <module name="javax.jws.api" export="true" optional="true" />
-        <module name="javax.mail.api" export="true" optional="true" />
-        <module name="javax.management.j2ee.api" export="true" optional="true" />
-        <module name="javax.persistence.api" export="true" optional="true" />
-        <module name="javax.resource.api" export="true" optional="true" />
+        <module name="jakarta.activation.api" export="true" optional="true" />
+        <module name="jakarta.annotation.api" export="true" optional="true" />
+        <module name="jakarta.batch.api" export="true" optional="true" />
+        <module name="jakarta.ejb.api" export="true" optional="true" />
+        <module name="jakarta.enterprise.api" export="true" optional="true" />
+        <module name="jakarta.enterprise.concurrent.api" export="true" optional="true" />
+        <module name="jakarta.inject.api" export="true" optional="true" />
+        <module name="jakarta.interceptor.api" export="true" optional="true" />
+        <module name="jakarta.json.api" export="true" optional="true" />
+        <module name="jakarta.jms.api" export="true" optional="true" />
+        <module name="jakarta.xml.ws.api" export="true" optional="true" />
```

Comment:
@yersan This is a nit for follow-up later, unless you touch this PR for some other reason.  This line can be removed as it is a duplicate. I assume it was created by sed converting javax.jws.api -> jakarta.xml.ws.api. I had the same problem in full WF.

Reply:
Will be tracked at https://issues.redhat.com/browse/WFLY-18071